chapters/operatorsandexpressions.tex

```diff
@@ -199,37 +199,38 @@ \section{Arithmetic Operators}\label{arithmetic-operators}
 
 Modelica supports five binary arithmetic operators that operate on any numerical type:
 \begin{center}
-\begin{tabular}{c|l}
-\tablehead{Operator} & \tablehead{Description} \\
+\begin{tabular}{c|l|l}
+\tablehead{Operator} & \tablehead{Description} & \tablehead{Reference} \\
 \hline
 \hline
-{\lstinline!^!} & Exponentiation\\
-{\lstinline!*!} & Multiplication\\
-{\lstinline!/!} & Division\\
-{\lstinline!+!} & Addition\\
-{\lstinline!-!} & Subtraction\\
+{\lstinline!+!} & Addition & \cref{addition-subtraction-and-string-concatenation}\\
+{\lstinline!-!} & Subtraction & \cref{addition-subtraction-and-string-concatenation}\\
+{\lstinline!*!} & Multiplication & \cref{array-element-wise-multiplication} and \cref{matrix-and-vector-multiplication-of-numeric-arrays}\\
+{\lstinline!/!} & Division & \cref{division-by-numeric-scalars} and \cref{array-element-wise-division}\\
+{\lstinline!^!} & Exponentiation & \cref{element-wise-exponentiation} and \cref{scalar-exponentiation-of-matrices}\\
 \hline
 \end{tabular}
 \end{center}
 
-Some of these operators can also be applied to a combination of a scalar type and an array type, see \cref{scalar-vector-matrix-and-array-operator-functions}.
+The semantics of these operators are given in \cref{scalar-vector-matrix-and-array-operator-functions} (both when applied to scalar operands and when one or both operands are of array type).
+For arrays this explains both the normal operator and the element-wise operators that start with a dot.
 
 The syntax of these operators is defined by the following rules from the Modelica grammar:
 \begin{lstlisting}[language=grammar]
 arithmetic-expression :
    [ add-operator ] term { add-operator term }
 
 add-operator :
-   "+" | "-"
+   "+" | "-" | ".+" | ".-"
```

Comment:
Can't we just start phasing out all these hard-to-maintain fragments of repeated grammar?  A cross reference or two should be enough, if we should keep anything at all.